### PR TITLE
Use HTML form-related elements semantically

### PR DESCRIPTION
**Pull request by rwd**

### CODE REVIEW

File: assets/scss/dropdown.scss [hunk 1]
```diff
@@ -31,10 +31,12 @@
     text-decoration: none;
   }
 
-  &.more-facets {    
-    strong {
+  &.more-facets {
+    legend {
       font-size: 1.5rem;
       font-weight: 300;
```
Comment (mirjamverloop) [inline assets/scss/dropdown.scss]: The font-weight can be removed as a legend has a font-weight of normal/300 by default.

File: assets/scss/dropdown.scss [hunk 2]
```diff
@@ -136,15 +138,15 @@
             padding: 0.25rem 1rem;
           }
 
-          input[type="checkbox"] {
+          input[type="checkbox"],input[type="radio"] {
```
Comment (mirjamverloop) [inline assets/scss/dropdown.scss]: For consistency (and readability) with other CSS files, move `input[type="radio"]` to the next line.

File: assets/scss/dropdown.scss [hunk 3]
```diff
@@ -136,15 +138,15 @@
...
             left: 0;
             opacity: 0;
             position: fixed;
             top: 0;
             z-index: -10;
           }
 
-          input[type="checkbox"]:checked + label { 
+          input[type="checkbox"]:checked + label,input[type="radio"]:checked + label {
```
Comment (mirjamverloop) [inline assets/scss/dropdown.scss]: See previous comment, move `input[type="radio"]:checked + label` to the next line.

File: components/search/MoreFiltersDropdownFacet.vue [hunk 4]
```diff
@@ -31,61 +31,59 @@
...
         isActive: false,
         limitTo: 9
       };
     },
-    computed: {
-      showMoreOrLess() {
-        if (this.isActive) {
-          return this.$t('facets.button.showLess', { label: this.$tc(`facets.${this.name}.name`, 2).toLowerCase() });
-        } else {
-          return this.$t('facets.button.showAll', { label: this.$tc(`facets.${this.name}.name`, 2).toLowerCase() });
```
Comment (lbiedinger) [inline components/search/MoreFiltersDropdownFacet.vue]: these .toLowerCase statements need to come across on the refactor.

Comment (rwd) [reply components/search/MoreFiltersDropdownFacet.vue]: Indeed. That's been restored.

**Pull request merged**